conf/base.config

```diff
@@ -60,6 +60,18 @@ process {
   withName:Bed12ToBigBed {
     if ({ task.exitStatus in [255] }) { errorStrategy = 'ignore' }
   }
+  withName:StringTie2 {
+    container = 'quay.io/biocontainers/stringtie:2.0--hc900ff6_0'
+  }
+  withName:StringTie2Merge {
+    container = 'quay.io/biocontainers/stringtie:2.0--hc900ff6_0'
+  }
+  withName:FeatureCounts {
+    container = 'quay.io/biocontainers/subread:2.0.1--hed695b0_0'
+  }
+  withName:Bambu {
+    container = 'docker.io/cheny1/bambu:0.3'
+  }
```

Comment:
Add all of this software to `environment.yml`

Reply:
for others, we can add it to environment.yml, but for bambu, there is no existing software published any where yet(in fact we plan to submit to bioconductor later), so what should we do with that?

Reply:
Ok. You can leave that in there then 👍 The container just needs to be tagged properly by release for reproducibility.